src/App.test.jsx

```diff
@@ -2,8 +2,13 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
+// hacky workaround to allow findBy to work
+// TODO: Figure out a cleaner solution..
+window.MutationObserver = require('mutation-observer');
+
 test('renders home page by default', () => {
   render(<App />);
-  const linkElement = screen.getByText(/Scholarships/i);
-  expect(linkElement).toBeInTheDocument();
+
+  const linkElement = screen.findByText(/Scholarships/i);
+  return expect(linkElement).resolves.toBeInTheDocument();
```

Comment:
So this file's changes are a bit unintuitive. Basically what's going on is that the home page gets updated in `useEffect` due to an auth state change so I've returned a promise here to wait on even though it's not quite related to that effect :/.